AFE/src/app/users/users.component.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Please remove spec.ts files
Also, when you create component with CLI, you could type something like this 
ng g c component-name --skipTests or -skip-tests